pkg/engine/networksecuritygroups.go

```diff
@@ -10,11 +10,19 @@ import (
 )
 
 func CreateNetworkSecurityGroup(cs *api.ContainerService) NetworkSecurityGroupARM {
+	var securityRules []network.SecurityRule
+	var sshRule, kubeTLSRule network.SecurityRule
+
 	armResource := ARMResource{
 		APIVersion: "[variables('apiVersionNetwork')]",
 	}
 
-	sshRule := network.SecurityRule{
+	if cs.Properties.MasterProfile != nil && len(cs.Properties.MasterProfile.NetworkSecurityRules) > 0 {
+		securityRules = cs.Properties.MasterProfile.NetworkSecurityRules
+		goto NSG
```

Comment:
I would prefer avoiding the use of goto here, can we have an `else` statement instead?